packages/vue/src/components/sign-up.vue

```diff
@@ -9,7 +9,7 @@
           {{ signUpButtonText }}
         </base-heading>
         <federated-sign-in></federated-sign-in>
-        <base-field-set :disabled="state.matches('signUp.submit')">
+        <base-field-set :disabled="actorState.matches('signUp.submit')">
```

Comment:
Federated sign in buttons on the sign up are not working for Vue or React. I get this error. I believe the federatedSignIn service doesn't exist in sign up.

```suggestion
        // add this into signUp services or call the one in SignIn actor
        
      async federatedSignIn(_, event) {
        const { provider } = event.data;
        const result = await Auth.federatedSignIn({ provider });

        return result;
      },
```

`Warning: No service found for invocation 'federatedSignIn' in machine 'signUpActor'.`

Reply:
Thanks for the catch -- https://github.com/aws-amplify/amplify-ui/pull/180/commits/b18cafa6cb49f0dd0ce81d8fe9280a78e6f32842. Validated locally 👍